tools/buildsys/src/main.rs

```diff
@@ -155,6 +161,10 @@ fn build_package() -> Result<()> {
         }
     }
 
+    if let Some(mods) = manifest.go_mods() {
+        GoMod::vendor(&root_dir, &manifest_dir, &mods).context(error::GoModSnafu)?;
+    }
```

Comment:
Need to emit `cargo:rerun-if-changed` directives for the output files, or tarball if we switch to that.